tests/post_training/pipelines/lm_weight_compression.py

```diff
@@ -164,25 +185,16 @@ def collect_data_from_stdout(self, stdout: str):
     def save_compressed_model(self) -> None:
         if self.backend == BackendType.FP32:
             return
-        ov.serialize(self.model, self.output_model_dir / self.OV_MODEL_NAME)
-        self.model_hf._save_config(self.output_model_dir)
+        if self.backend == BackendType.TORCH:
+            self.model_hf.save_pretrained(self.output_model_dir)
```

Comment:
To export PyTorch HF model to openvino you can use `from optimum.exporters.openvino.convert import export_from_model`

I suggest to save compressed PyTorch model and FP32 PyTorch model in OV IR.
```suggestion
        if self.backend == BackendType.OV:
            ov.serialize(self.model, self.output_model_dir / self.OV_MODEL_NAME)
            self.model_hf._save_config(self.output_model_dir)
        elif self.backend == BackendType.TORCH:
            export_from_model(self.model_hf, self.output_model_dir, stateful=False, compression_option="fp32")
```

Reply:
Done, thank you